maps/site53/job/outfits.dm

```diff
@@ -767,16 +797,16 @@
 	glasses = /obj/item/clothing/glasses/night
 	gloves = /obj/item/clothing/gloves/thick/combat
 	shoes = /obj/item/clothing/shoes/combat
-	id_types = list(/obj/item/card/id/adminlvl4)
-	suit_store = /obj/item/gun/launcher/grenade // LEEEET'S DO IIIT
-	r_hand = /obj/item/clothing/accessory/storage/bandolier
+	id_types = list(/obj/item/card/id/physics)
+	suit_store = null
+	r_hand = /obj/item/gun/launcher/grenade // LEEEET'S DO IIIT
```

Comment:
b r u h

Reply:
Eh, hell, ERTs ought to feel strong. It's not like all of security don't have access to automatic weapons